File: java/oop/CircleApp/src/com/techlabs/circle/Circle.java
package com.techlabs.circle;

public class Circle {
	private static final float PI = 3.14f;
	float radius;

	
	
	private BorderOptions option;
	
	public Circle(float d, BorderOptions option ){
		
		this.option = option;
		this.radius = d;
	}
	
	public double getDouble(){
		return this.radius;
	}
	
	public BorderOptions getBorderOption(){
		return this.option;
	}
	
	public double calculateArea(){
		System.out.println("Area of Circle is: "+(PI*radius*radius));
		return (PI*radius*radius);
	}


}

File: dotNet/csharp/CircleApp/CircleApp/Circle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CircleApp
{
    class Circle
    {
        private  const float _Pi = 3.14f;
        private float _radius;
        private BorderOptions _borderOption;

        public Circle(float d, BorderOptions option)
        {

            this._borderOption = option;
            this._radius = d;
        }

        public float Radius
        {
            get
            {
                return _radius;
            }
        }

        public BorderOptions BorderOption
        {
            get
            {
                return _borderOption;
            }
        }

        public double CalculateArea()
        {
            return (_Pi*_radius*_radius);
        }


           



    }
}
